lib/AST/Attr.cpp

```diff
@@ -1073,8 +1073,11 @@ const AvailableAttr *AvailableAttr::isUnavailable(const Decl *D) {
     return attr;
 
   // If D is an extension member, check if the extension is unavailable.
-  if (auto ext = dyn_cast<ExtensionDecl>(D->getDeclContext()))
-    return AvailableAttr::isUnavailable(ext);
+  //
+  // Skip decls imported from Clang: rdar://problem/53956555
```

Comment:
Can you add more detail here for people outside of Apple?

Reply:
I've added some details, and the test should help to understand the problem too.